packages/mux-player/src/index.ts

```diff
@@ -512,13 +508,13 @@ class MuxPlayerElement extends VideoApiElement implements MuxPlayerElement {
           window.requestAnimationFrame(chromeWorkaround);
         }
       };
-      mc?.media?.addEventListener('playing', () => {
+      this.addEventListener('playing', () => {
         chromeWorkaround();
       });
     }
 
-    mc?.addEventListener('userinactivechange', () => {
-      const newUserInactive = mc?.hasAttribute('user-inactive');
+    this.addEventListener('userinactivechange', () => {
```

Comment:
the thing is that the media controllers are blasted away each time the theme changes, and since the event bubbles up.
event delegation seemed the easiest here

Reply:
ah, didn't realize that it bubbled up. If it works, I don't have any issues.